aws-datastore/src/main/java/com/amplifyframework/datastore/DataStoreConfiguration.java

```diff
@@ -206,4 +291,48 @@ public DataStoreConfiguration build() {
             );
         }
     }
+
+    enum ConfigKey {
+        /**
+         * At most one base sync will be performed within this interval of time.
+         * The interval is expressed in milliseconds.
+         */
+        SYNC_INTERVAL("syncInterval"),
+        /**
+         * Number of items requested per page in sync operation.
+         */
+        SYNC_PAGE_SIZE("syncPageSize"),
+        /**
+         * Number of records that the client wants to process, while it is requesting
+         * a base/delta sync operation from AppSync.
+         */
+        SYNC_MAX_RECORDS("syncMaxRecords");
+
+        private final String key;
+
+        ConfigKey(String key) {
+            this.key = key;
+        }
+
+        @NonNull
+        @Override
+        public String toString() {
+            return key;
+        }
+
+        /**
+         * Lookup a config key by its string value.
+         * @param anything Any string found in the key position of the plugin config JSON
+         * @return An enumerate config key
+         */
+        @SuppressWarnings("unused")
```

Comment:
I think this is from an earlier phase of this code's evolution, where this method was not being used, yet. But, it does get used now, up on L113. So we could remote this suppression.